project.clj

```diff
@@ -20,15 +21,19 @@
             [lein-cljsbuild "1.1.7" :exclusions [org.clojure/clojure
                                                  org.clojure/clojurescript]]
             [lein-figwheel "0.5.14" :exclusions [org.clojure/clojure
-                                                 org.clojure/clojurescript]]]
+                                                 org.clojure/clojurescript]]
+            [com.github.clj-kondo/lein-clj-kondo "0.2.4" :exclusions [org.clojure/clojure
+                                                                      org.clojure/clojurescript]]]
+  :aliases {"clj-kondo-deps" ["clj-kondo" "--copy-configs" "--dependencies" "--parallel" "--lint" "$classpath"]
+            "clj-kondo-lint" ["do" ["clj-kondo-deps"] ["clj-kondo" "--lint" "src/main" "--fail-level" "error"]]}
```

Comment:
Why not lint tests too?

Reply:
I'm going to spend some time trying to lint the tests (they have some errors unrelated to clara itself I think, just the result of having developed the code-base without linting it.

Reply:
@mrrodriguez  alright, so the problem was primarily that I had not added support for `def-rules-test`, I have done that now and everything is being linted with no errors.